### ISSUE TITLE
Are AdaPoinTr and PoinTr++ the same framework?

### ISSUE THREAD

**Issue by yorklyb** [yuxumin/PoinTr#96]
Thanks for the release of AdaPoinTr! I'm wondering if AdaPoinTr is previously known as PoinTr++ in the report of MVP challenge 2021, which was the winner of that challenge as well.

**Comment by yuxumin** [comment]
Hi, thanks for your interest in PoinTr ( PoinTr++) and AdaPoinTr. They are different work. 
PoinTr++ (PoinTr + Cascade Refiner) see here https://arxiv.org/pdf/2112.12053.pdf
AdaPoinTr (PoinTr + adaptive denoising query design)
AdaPoinTr outperform PoinTr++ with less params and FLOPs, and can be trained end2end

**Comment by yorklyb** [comment]
So does AdaPoinTr outperform PoinTr++ in the MVP completion task as well? Or the current version of AdaPoinTr is for the single-shot reconstruction task only?

**Comment by yuxumin** [comment]
The test set for the MVP completion is unavaliable, but AdaPoinTr outperforms PoinTr++ on the validation set. In my opinion, AdaPoinTr should consistently outperform PoinTr++ on MVP completion (test set for ICCVW competition)

**Comment by yorklyb** [comment]
Thanks for your reply!
